doc/source/serve/api/index.md

```diff
@@ -53,6 +53,7 @@ This is fixed by added custom filename mappings in `source/conf.py` (look for "a
 
    serve.get_replica_context
    serve.get_multiplexed_model_id
+   serve.get_app_handle
```

Comment:
add `get_deployment_handle` too

make sure the docstring clarifies that it is a developer API & any limitations

Reply:
Added and clarified in the docstring!